ex6/ex6.c

```diff
@@ -20,6 +20,21 @@
 int main()
 {
     // Your code here
+    struct timespec start, end;
+    uint64_t diff;
+
+    int total = 0;
+
+    for (int i = 0; i < number_iter; i++) {
+        clock_gettime(CLOCK_MONOTONIC, &start);
```

Comment:
Cool use of of Clock Monotonic. wondering what the difference between that and real time.